docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -30,15 +30,13 @@ and Machine ID](../../machine-id/deployment/spacelift.mdx).
 ## Step 1/3. Create Teleport credentials for Terraform
 
 Terraform needs a signed identity file from the Teleport cluster certificate
-authority to manage resources in the cluster.
-You can create a local Teleport user for this purpose or you can use the machine identity agent
-[(Machine ID)](../../machine-id/introduction.mdx) to generate credentials.
+authority to manage resources in the cluster. Terraform is intended to be used with
+[(Machine ID)](../../machine-id/introduction.mdx), so this tutorial will show you how to
+use a local Machine ID bot to try it out.
```

Comment:
I think this information about the scope of the guide (i.e., that it includes Machine ID instructions) belongs in the introduction of the guide.

Reply:
Updated!